template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/raz/RangerRazDatalakeConfigProvider.java

```diff
@@ -34,7 +34,8 @@ public Map<String, ApiClusterTemplateService> getAdditionalServices(CmTemplatePr
             Set<HostgroupView> hostgroupViews = source.getHostgroupViews();
 
             return hostgroupViews.stream()
-                    .filter(hg -> hg.getName().toLowerCase().equals("master"))
+                    .filter(hg -> hg.getName().toLowerCase().equals("master")
+                            || hg.getName().toLowerCase().equals("razhg"))
```

Comment:
please define a static string list in the class and check if that list contains `hg.getName`